internal/peer/chaincode/chaincode.go

```diff
@@ -115,18 +97,8 @@ func resetFlags() {
 		"The channel on which this command should be executed")
 	flags.StringVarP(&policy, "policy", "P", common.UndefinedParamValue,
 		"The endorsement policy associated to this chaincode")
```

Comment:
I believe these flags can also be removed since they are not used for invoke or query:
- lang
- path
- version
- username
- policy

Also remove the corresponding variables above.

Reply:
I do not think you can remove version, since it's part of the chaincode specification. I will remove the rest.

Reply:
@denyeart `lang`, `version` and `path` are used in `ChaincodeSpec`, hence we cannot take them away, while I removed `policy` and `username`. 

See:

![image](https://github.com/hyperledger/fabric/assets/3093578/ca5dd7eb-8d46-48ac-8368-2576d7c21cc4)